### PR TITLE
fix(auction): Only show preview message if eligibleSaleArtworks=0

### PR DESCRIPTION
**Pull request by damassi**
WIP

The type of this PR is: **Bugfix**

### Description

Instead of only showing a preview message if sale is in preview, also check to see the number of `eligibleSaleArtworks` in the sale. If `0`, show the preview message. 

cc @artsy/grow-devs

### CODE REVIEW

**Review by jacobherrington — COMMENTED** [review]
Looks like one spec needs to be updated to fix CI!

[src/v2/Apps/Auction/__tests__/AuctionApp.jest.tsx](https://github.com/artsy/force/blob/main/src/v2/Apps/Auctions/__tests__/AuctionsApp.jest.tsx)

File: src/v2/Apps/Auction/__tests__/AuctionApp.jest.tsx [hunk 1]
```diff
@@ -309,13 +325,21 @@ describe("AuctionApp", () => {
...
       })
       expect(wrapper.find("AuctionCurrentAuctionsRail").length).toBe(1)
     })
+
+    it("does not render the auction rail", () => {
+      const wrapper = getWrapper({
+        Sale: () => ({}),
+      })
+      expect(wrapper.find("AuctionCurrentAuctionsRail").length).toBe(0)
+    })
```
Comment (jacobherrington) [inline src/v2/Apps/Auction/__tests__/AuctionApp.jest.tsx:343]: I added this to make the CI pass, but the conditions seem backward to me? I was a little confused so I wanted to push to commit and ask.

Comment (damassi) [reply src/v2/Apps/Auction/__tests__/AuctionApp.jest.tsx:343]: Thank you!

**Comment by damassi** [comment]
superseded by revert of revert here: https://github.com/artsy/force/pull/10053

**Pull request closed without merging**